tests/task_queue_consumer/task_queue_consumer_test.py

```diff
@@ -78,9 +82,10 @@ def test_task_queue_consumer_private_subnet():
 
 @pytest.mark.unit
 @serialise_mocks()
-@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client)
+@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client.get_parameters)
 def test_task_queue_consumer_no_subnet():
-    task_queue_consumer.ssm_client.get_parameters.return_value = ssm_return_vals(False)
+    task_queue_consumer.ssm_client.get_parameters.return_value = ssm_return_vals(
```

Comment:
Line length changes here too. Lets please agree on a line length and avoid these